pandas/tests/reshape/test_concat.py

```diff
@@ -1243,6 +1244,44 @@ def test_concat_dict(self):
         expected = concat([frames[k] for k in keys], keys=keys)
         tm.assert_frame_equal(result, expected)
 
+    def test_concat_mapping(self):
+        class SomeMapping(Mapping):
```

Comment:
thanks @pbourguignon for the update. we have a fixture for this...non_mapping_dict_subclass

Reply:
> thanks @pbourguignon for the update. we have a fixture for this...non_mapping_dict_subclass

Sorry, missed that one. Will update accordingly.

Reply:
> thanks @pbourguignon for the update. we have a fixture for this...non_mapping_dict_subclass

I suspect it was actually meant to be called "non_dict_mapping_subclass"... Should I change it and fix all occurrences? Better in a separate PR?

Reply:
makes sense. normally a separate PR for unrelated changes, but since only a few occurrences could do it here.

Reply:
> makes sense. normally a separate PR for unrelated changes, but since only a few occurrences could do it here.

For the sake of others' awareness, I'll prepare a separate PR for that once this one is merged in.